### PR TITLE
Changes On Package.json file

### PR DESCRIPTION
**Pull request by williamdaconceicao**
Add the lint on the "yarn lint" command

### CODE REVIEW

**Comment by mikbry** [comment]
"yarn run lint" it lint src code not tests directory. Also the "run" is not mandatory. Try:
`eslint tests/shared --ext=js && jest`

File: package.json [hunk 1]
```diff
@@ -11,7 +11,7 @@
     "compile:css": "yarn clean-dist && cleancss -o dist/compressed.css node_modules/material-components-web/dist/material-components-web.css node_modules/dialog-polyfill/dialog-polyfill.css src/libs/css/app.css",
     "dev": "yarn compile:dev && yarn start:dev",
     "lint": "eslint src/libs --ext=js --ext=jsx && eslint src/demos --ext=js --ext=jsx",
-    "test": "jest",
+    "test": "yarn run lint && yarn run jest",
```
Comment (willdurand) [inline package.json]: no, test should be test. Fix the travis ci config if you want to run both commands.

Comment (mikbry) [reply package.json]: Ok I agree, but should we add it to lint command ?

Comment (willdurand) [reply package.json]: because you want a command to run both at the same time? maybe "check" then

**Pull request closed without merging**